.ruby-version

```diff
@@ -0,0 +1 @@
+3.0.1
```

Comment:
Why do we need ruby related configuration here?

Reply:
The reason is Ruby is needed, is to install cocoapods and ffi, used for iOS. More information: because `flutter run` can run in the root directory, the ruby version should be specified wherever this can be run, not just the iOS directory. We actually already have it, but in an uncommon format: it's in `.tool-versions`. 

However, if want to de-duplicate `3.0.1`, we should delete `tool-versions` because that is a niche format only used by `asdf`, where as `asdf` can also use `ruby-version`.

Reply:
Pretty detailed! Thanks :)

Reply:
@ben-xD It was already specified in [.tool-versions](https://github.com/ably/ably-flutter/blob/main/.tool-versions). It's not ideal to maintain this in two places, but if you must then please update both.

Reply:
`.tool-version` was already deleted because ASDF can read `.ruby-version`, so there is no duplication. That file was deleted in 5bae664

Reply:
ok, sorry, I looked at the wrong branch. I also didn't realise that ASDF would read `.ruby-version`. I can think of at least [one other place](https://github.com/ably/ably-asset-tracking-swift/blob/main/.tool-versions) we could make this switch.

Reply:
Okay good idea, I made that change: https://github.com/ably/ably-asset-tracking-swift/pull/175